frontend/portals/SmileContainer/connector.js

```diff
@@ -9,16 +9,8 @@ import { mountSmileScript } from '../../actions';
 const mapStateToProps = state => ({
   digest: getSmileDigest(state),
   externalCustomerId: getExternalCustomerId(state),
+  initializeScript: (externalCustomerId, digest) =>
```

Comment:
you don't map any state to the props here. You should directly import the helper in the component